bugsnag-android-core/src/main/java/com/bugsnag/android/ExceptionHandler.java

```diff
@@ -66,6 +68,10 @@ public void uncaughtException(@NonNull Thread thread, @NonNull Throwable throwab
                     metadata, severityReason, null);
         }
 
+        forwardToOriginalHandler(thread, throwable);
```

Comment:
I think this leaves us open to a death loop if anything in `uncaughtException` throws an exception. What about a try/finally (with forwarding code in finally) wrapping an inner try/catch that discards anything that goes wrong in our handler code?

Reply:
We *might* be, although the docs on `uncaughtException` clearly state:
> Any exception thrown by this method will be ignored by the Java Virtual Machine.

https://developer.android.com/reference/java/lang/Thread.UncaughtExceptionHandler#uncaughtException(java.lang.Thread,%20java.lang.Throwable)

Reply:
Maybe it's just because I'm getting older, but I never trust API promises :P

Another benefit would be a single point where it calls `forwardToOriginalHandler`, no matter how many return points we have.